src/malli/malli.cljc

```diff
@@ -0,0 +1,302 @@
+(ns malli.malli)
+
+(set! *warn-on-reflection* true)
+
+;;
+;; protocols
+;;
+
+(defprotocol IntoSchema
+  (-id [this])
```

Comment:
Is `id` good name? Could it be `name` or even `keyword` (which is used in the json-schema)/

Reply:
it's `name` now.